app/utils/stacking/pox.spec.ts

```diff
@@ -0,0 +1,92 @@
+import {
+  makeRandomPrivKey,
+  getAddressFromPrivateKey,
+  TransactionVersion,
+} from '@blockstack/stacks-transactions';
+import { POX } from './pox';
+import { Api } from '../../api/api';
+import BN from 'bn.js';
+import * as bitcoin from 'bitcoinjs-lib';
+
+const client = new POX();
+const api = new Api('http://localhost:3999');
+
+const btcAddress = '17VZNX1SN5NtKa8UQFxwQbFeFc3iqRYhem';
+
+const waitForTxConfirm = (txid: string) => {
+  return new Promise((resolve, reject) => {
+    const interval = setInterval(() => {
+      const getTX = async (interval: number) => {
+        try {
+          const txResponse = await api.getTxDetails(txid);
```

Comment:
This could result in aggressive / redundant network requests if these happen to take longer the timer's 500ms interval